packages/augur-ui/src/modules/portfolio/components/portfolio-view/my-bets.tsx

```diff
@@ -15,31 +19,62 @@ import {
   MY_BETS_BET_DATE,
   GAMES,
   FUTURES,
+  EVENT,
+  OUTCOMES,
+  MARKET_STATE_TYPES,
 } from 'modules/common/constants';
 import { MARKETS } from 'modules/routes/constants/views';
 import { FilterNotice } from 'modules/common/filter-notice';
 import { EmptyMagnifyingGlass } from 'modules/common/icons';
-import { MOCK_GAMES_DATA, MOCK_FUTURES_DATA } from 'modules/trading/store/constants';
-import { Game } from '../common/common';
+import {
+  MOCK_GAMES_DATA,
+  MOCK_FUTURES_DATA,
+  MOCK_OUTCOMES_DATA,
+} from 'modules/trading/store/constants';
+import { Game, Outcomes } from '../common/common';
 
 export const MyBets = () => {
   const [state, setState] = useState({
-    selectedMarketCardType: 0,
+    selectedMarketCardType: SPORTS_MARKET_TYPES[0].id,
     viewBy: MY_BETS_VIEW_BY[0].value,
     marketStatus: MY_BETS_MARKET_STATUS[0].value,
     betDate: MY_BETS_BET_DATE[0].value,
     rows: MOCK_GAMES_DATA,
+    selectedMarketStateType: MARKET_STATE_TYPES[0].id,
   });
 
-  const { selectedMarketCardType, viewBy, marketStatus, betDate, rows } = state;
+  const {
+    selectedMarketCardType,
+    viewBy,
+    marketStatus,
+    betDate,
+    rows,
+    selectedMarketStateType,
+  } = state;
+
+  useEffect(() => {
+    if (MY_BETS_VIEW_BY[viewBy].label === EVENT) {
+      setState({ ...state, rows: MOCK_GAMES_DATA });
+    } else if (MY_BETS_VIEW_BY[viewBy].label === OUTCOMES) {
+      setState({ ...state, rows: MOCK_OUTCOMES_DATA });
+    }
+  }, [viewBy]);
 
   useEffect(() => {
-    if (SPORTS_MARKET_TYPES[selectedMarketCardType].label === GAMES) {
-      setState({...state, rows: MOCK_GAMES_DATA})
-    } else if (SPORTS_MARKET_TYPES[selectedMarketCardType].label === FUTURES) {
-      setState({...state, rows: MOCK_FUTURES_DATA})
-    } 
-  }, [selectedMarketCardType]);
+    if (
+      MY_BETS_VIEW_BY[viewBy].label === EVENT &&
+      SPORTS_MARKET_TYPES[selectedMarketCardType].label === GAMES
+    ) {
+      setState({ ...state, rows: MOCK_GAMES_DATA });
+    } else if (
+      MY_BETS_VIEW_BY[viewBy].label === EVENT &&
+      SPORTS_MARKET_TYPES[selectedMarketCardType].label === FUTURES
+    ) {
+      setState({ ...state, rows: MOCK_FUTURES_DATA });
+    }
+  }, [selectedMarketCardType, viewBy]);
+
+  const showEvents = MY_BETS_VIEW_BY[viewBy].label === EVENT;
```

Comment:
should define this higher and reference it in the use effects so that we don't have to keep repeating it